catalog/core/catalog-core-impl/filter-proxy/src/test/java/ddf/catalog/filter/proxy/builder/test/FilterBuilderTest.java

```diff
@@ -552,208 +552,4 @@ public Object visit(Within filter, Object data) {
         },
         null);
   }
-
-  // @Test
-  public void test() {
```

Comment:
I'm not sure if this test should be removed. Looks like it's been inactive since the initial commit. I don't know if these lines provide any value, like showing examples of how to build filters.

Reply:
We have version control so we can always get it back if we need it.